CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.0.1]
+### Fixed
+- Request validation should not throw if verifyingContract is not defined in typed signature ([#328](https://github.com/MetaMask/eth-json-rpc-middleware/pull/328))
```

Comment:
Nit: Is this a better description? The field is in the message, not the message type or the signature for the message.
```suggestion
- Request validation should not throw if verifyingContract is not defined in typed message ([#328](https://github.com/MetaMask/eth-json-rpc-middleware/pull/328))
```